README.md

```diff
@@ -1 +1,83 @@
-Placeholder for Checkstyle Regression Tool readme.
+# Regression-tool
+
+Regression-tool is an automation tool to do regression testing based on proposed patch(Pull Request) 
+of checkstyle project.
+
+## Setup

```

Comment:
We need to mention all requirements.
We need contribution repo already cloned, we need checkstyle repo already cloned.

I would also like to see steps. It is not clear that each category is required to do, imo.